src/main/features/dashboard/views/macro/claimStatus/claimFullyAdmitted.njk

```diff
@@ -0,0 +1,111 @@
+{% from "externalLink.njk" import externalLink %}
+{% from "form.njk" import linkButton %}
+
+{% macro claimFullAdmissionForDefendantDashboard(claim) %}
+
+  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    {{ t('You’ve admitted all of the claim and said you’ll pay the full amount immediately.') }}
+  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+    {% set paymentMethod = 'by ' + claim.response.paymentIntention.paymentDate | date %}
```

Comment:
Please move `by ` into main string otherwise i18n will not work properly. A `paymentMethod` should be really removed now.